### DISCUSSION TITLE
Snakemake pipeline?

### DISCUSSION THREAD

**Discussion by rargelaguet** [GreenleafLab/ArchR#1066] [Questions / Documentation]
Hello,
I would like to set up a Snakemake pipeline for scATAC-seq analysis using ArchR. It would be useful if ArchR were uploaded to conda (as Signac, https://anaconda.org/bioconda/r-signac), as this facilitates dependency management.

P.S. By any chance do the authors have a Snakemake pipeline that are willing to share? 

Thanks,
Ricard

**Comment by rcorces** [comment]
We do not have a snakemake. Every analysis is different and I dont think it should be pipelined. In fact, one of the main goals of ArchR is to be able to do interactive analysis.

Unfortunately, I dont think we will be going the conda route but we are working on releasing both a containerized version of ArchR as well as an renv environment (https://rstudio.github.io/renv/articles/renv.html). We are just extremely bandwidth limited at the moment.

**Comment by rargelaguet** [comment]
ok, thanks for your quick reply!
